### PR TITLE
code actualisation

### PR DESCRIPTION
**Pull request by AlexeyBudynkov**
* Changed writeToPlistFile procedure
* Changed failed tests

### CODE REVIEW

File: iOSDeviceManager/Utilities/XCTestConfigurationProxy.m [hunk 1]
```diff
@@ -100,28 +134,21 @@ - (BOOL)writeToPlistFile:(NSString *)path overwrite:(BOOL)overwrite {
         }
     }
 
-    Class klass = [self.configuration class];
-    SEL selector = NSSelectorFromString(@"writeToFile:");
+//    NSDictionary *plist;
```
Comment (sergei-fedorov) [inline iOSDeviceManager/Utilities/XCTestConfigurationProxy.m]: Let's get rid of the commented code?

**Review by sergei-fedorov — COMMENTED** [review]
Is it expected that some ci builds failed?

File: iOSDeviceManager/Utilities/XCTestConfigurationProxy.m (hunk 1 above)
Comment (AlexeyBudynkov) [reply iOSDeviceManager/Utilities/XCTestConfigurationProxy.m]: > Let's get rid of the commented code?

Fixed in the next commit.

**Comment by AlexeyBudynkov** [comment]
> Is it expected that some ci builds failed?

I think something has broken for Xcode-11.7 and Xcode-12.0 because of private headers changing. Should we continue to support these Xcode versions?

@IlyaBausovAkvelon could you, please, check this?

**Review by IlyaBausovAkvelon — APPROVED** [review]

**Pull request merged**